VOKMockUrlProtocol.m

```diff
@@ -111,6 +118,28 @@ - (NSArray *)resourceNames
         [resourceName appendFormat:queryFormat, self.request.URL.query];
     }
     
+    if (headersToEncode) {
+        NSDictionary *headerDict = self.request.allHTTPHeaderFields;
```

Comment:
There's a potential pitfall here in that, at least according to Apple's docs, header names are case-sensitive, but my experience has been that the majority of servers treat them as case-insensitive.  Basically, if the app sets `AUTHORIZATION: whatever` and I ask for the `Authorization` header, it won't find it.

I'd be tempted to iterate over `headerDict`, normalize the header name to lower or upper case and compare to a normalized version of `headersToEncode`.  I'm not sure that's a good idea, though—it might be better to just document the potential pitfall.